### ISSUE TITLE
FloatingToolbal - Accessibility 

### ISSUE THREAD

**Issue by dratwas** [wordpress-mobile/gutenberg-mobile#1923]
**Describe the bug**
During fixing UI tests for WP-iOS that was related to accessibility i found an issue that the FloatingToolbar is not available by accessibility (can not click button "navigate up" with VoiceOver). It's because the whole `FlaotingToolbar` component is wrapped into `TouchableWithoutFeedback`. Because of that when the `FloatingToolbal` is focused and VoiceOver mode is ON, it tries to click the `TouchableWithoutFeedback` instead of i.e "Navigate Up" button.

**To Reproduce**
1. Turn on the VoiceOver
2. Add Paragraph block
3. Select that Paragraph block
4. Focus the FloatingToolbar and try to call the action on it
5. Nothing happens

**Expected behavior**
- The Navigate Up button should be accessible
- Breadcrumbs should be accessible

**Comment by pinarol** [comment]
I haven't checked the code yet but once the block is selected we should set accessible=false in the container TouchableWithoutFeedback so the child views become accessible.
